### PR TITLE
EHUB-227 Contact Us feature

### PR DESCRIPTION
**Pull request by nshaw**

### CODE REVIEW

**Comment by nshaw** [comment]
https://github.com/nshaw/2022-06-hub-application-bundle is a test bundle with the updated functionality. Not perfect but I think it's a decent candidate for MVP.

File: application/ui/widgets/eh-widgets-dir/eh-widgets/src/i18n/locales/it.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
       }
     },
     "bundleGroupInfo": {
+      "contactUsInfo": "Si prega di contattare l'autore per ulteriori informazioni",
```
Comment (nshaw) [inline application/ui/widgets/eh-widgets-dir/eh-widgets/src/i18n/locales/it.json:20]: @GermanoGiudici no need to review this entire PR but if you can review these translations I could use an Italian eye on them. I can show you the context when we talk.

File: application/ui/widgets/eh-widgets-dir/eh-widgets/src/i18n/locales/it.json [hunk 2]
```diff
@@ -48,6 +49,8 @@
       "name": "Nome",
       "category": "Categoria",
       "documentAddress": "Indirizzo della documentazione",
+      "displayContactLink": "Visualizza il pulsante Contattaci?",
```
Comment (GermanoGiudici) [inline application/ui/widgets/eh-widgets-dir/eh-widgets/src/i18n/locales/it.json:52]: @nshaw  remove question mark

File: application/ui/widgets/eh-widgets-dir/eh-widgets/src/i18n/locales/it.json [hunk 3]
```diff
@@ -125,6 +129,8 @@
     "nameReq": "Il nome è un campo obbligatorio",
     "min3Char": "Il nome deve contenere almeno 3 caratteri",
     "max25Char": "Il nome non deve superare i 25 caratteri",
+    "contactUrlFormat": "Fornisci un URL di contatto valido",
+    "contactUrlRequired": "Il URL di contatto è un campo obbligatorio",
     "docRequired": "La documentazione è un campo obbligatorio",
     "docFormat": "La documentazione deve essere fornita tramite un URL",
```
Comment (GermanoGiudici) [inline application/ui/widgets/eh-widgets-dir/eh-widgets/src/i18n/locales/it.json:135]: @nshaw "La documentazione deve essere fornita tramite una URL"

File: application/ui/widgets/eh-widgets-dir/eh-widgets/src/i18n/locales/it.json (hunk 2 above)
Comment (GermanoGiudici) [reply application/ui/widgets/eh-widgets-dir/eh-widgets/src/i18n/locales/it.json:52]: "Visualizza il pulsante Contattaci"

**Pull request merged**